capstone-backend/src/main/java/com/google/sps/servlets/ClubServlet.java

```diff
@@ -69,4 +69,11 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
       response.getWriter().println(gson.toJson(updatedClub));
     }
   }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    boolean successfulDelete = Utility.delete(clubs, request, response);
+    response.setContentType("application/json;");
+    response.getWriter().println(gson.toJson(successfulDelete));
```

Comment:
As written it will, but I'll change it to send a `204` instead. In the same vein, I'll remove lines 76 and 77 as now all handling is done in the Utility class. If the object is successfully deleted, Utility will send a 204. If it is not, some error code will be sent.